src/libopenrave-core/jsonparser/jsonreader.cpp

```diff
@@ -35,6 +35,45 @@
 
 namespace OpenRAVE {
 
+static int64_t ConvertIsoFormatDateTimeToLinuxTimeUS(const char* pIsoFormatDateTime)
+{
+    if (pIsoFormatDateTime == nullptr) {
+        return 0;
+    }
+    // RFC 3339 Nano format (2006-01-02T15:04:05.999999999Z07:00)
+    struct tm datetime;
+    const char *remain = strptime(pIsoFormatDateTime, "%FT%T", &datetime);
+    if (remain == nullptr) {
+        return 0;
```

Comment:
that's not correct

Reply:
Not sure what you mean it’s not correct. If you prefer !! check I can change it.